regions_www/index.html

```diff
@@ -61,7 +61,7 @@
           <div class="main-container">
             <p> Welcome to the Propensity to Cycle Tool (PCT) for England and Wales, which provides an evidence base to inform cycling investment.</p>
             <p>To run the PCT, first hover over a region on the map on the left to see the regional potential, then click on the region.</p>
-            <p>For an introduction to the tool, its use, and the scenarios, please watch the video above.</p>
+            <p>For an introduction to the tool, its use, and the scenarios, please watch the video above and read the <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a>.</p>
```

Comment:
Sorry to be a bit of a pain but I'd prefer this to be

https://npct.github.io/pct-shiny/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf

as Github pages are designed as a CDN whereas the `cdn.rawgit.com` is a bit of a hack and might change.